gp-includes/template.php

```diff
@@ -582,3 +582,60 @@ function gp_entry_actions( $seperator = ' &bull; ' ) {
 	<a href="#" class="gtranslate" tabindex="-1"><?php _e( 'Translation from Google', 'glotpress' ); ?></a>
 	*/
 }
+
+/**
+ * Returns the warning class for the translation row.
+ *
+ * @param Translation_Entry $t translation entry for the row.
+ *
+ * @since 2.2.0
+ *
+ * @return string
+ */
+function gp_get_translation_row_warning_class( $t ) {
+	$warning_class = $t->warnings? 'has-warnings' : 'no-warnings';
+	return $warning_class;
+}
+
+
+/**
+ * Generate a list of classes to be added to the translation row, based on translation entry properties.
```

Comment:
Docs are third-person singular elements, so third-person singular verbs should be used. (https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#language)